bin.src/genDefectRegistry.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, path, version, validStart, validEnd=None):
     # ISO-8601 will sort just fine without conversion from str
     rowList.sort(key=lambda row: row.validStart)
     for thisRow, nextRow in zip(rowList[:-1], rowList[1:]):
-        thisRow.validEnd = (datetime.datetime.strptime(nextRow.validStart, "%Y-%m-%d") -
+        thisRow.validEnd = (datetime.datetime.strptime(nextRow.validStart, "%Y-%m-%d") - # noqa
```

Comment:
Why does this need to be exempt from linting?

Reply:
I ran into a large number of linting errors when the flake8 version (I think) was changed at the end of October, and exempted all the troublesome lines to avoid the issue.  I'd forgotten about this, but have now removed the unnecessary `noqa` comments.